validator/delphi_validator/errors.py

```diff
@@ -31,6 +31,7 @@ def __init__(self, check_data_id, expression, message):
             pass a check, provide the date
             - message: str explaining why an error was raised
         """
+        super().__init__(message)
```

Comment:
The super init changes the output of `Validator.exit()` so that only the error message is printed. This isn't a concern here since you have the incoming validator report PR; just wanted you to be aware that the goal is to include the  `check_data_id` (unique error-data combo identifier), expression, and message in error output.

Reply:
I thought I had fixed this by moving the call to `super().__init__()` to the top of the function, but you are correct that it squashes the full error message.  I have changed this to take all the arguments which produces the same output as before.